server/src/main/scala/services/exports/flights/ArrivalToCsv.scala

```diff
@@ -16,7 +16,7 @@ object ArrivalToCsv {
       arrival.flightCodeString,
       arrival.Origin.toString,
       arrival.Gate.getOrElse("") + "/" + arrival.Stand.getOrElse(""),
-      arrival.displayStatus.description,
+      arrival.displayStatus(false).description,
```

Comment:
It would be nicer to create new function for mobile rather than passing a boolean, ie `displayStatusMobile`